### PR TITLE
DOC: Fix examples in pandas/core/ops/

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @MomIsBestFriend 

I think in a follow up would be good to have a single call for all the doctests. Not sure why we group few cases, then call some files individually...

Also, if you keep fixing tests, may be you can fix one directory at a time. So, for example, instead of having to call evey file in `pandas/core/arrays` separetely, we can just have one call for the whole directory.

Ideally, at some point we simply want to have a call to `pytest -q --doctest-modules pandas`

Thanks!

**Comment by MomIsBestFriend** [comment]
> lgtm, thanks @MomIsBestFriend 
> 
> I think in a follow up would be good to have a single call for all the doctests. Not sure why we group few cases, then call some files individually...
> 
> Also, if you keep fixing tests, may be you can fix one directory at a time. So, for example, instead of having to call evey file in `pandas/core/arrays` separetely, we can just have one call for the whole directory.
> 
> Ideally, at some point we simply want to have a call to `pytest -q --doctest-modules pandas`
> 
> Thanks!

@datapythonista Exactly, this is what I am aiming for.

And about the reason why there are "groupings" of, some of them are there for legacy reasons and the other reason is to have it as a "placeholder", I was afraid that even if a file is fixed, overtime it will become "wrong" agian, so this is just putting things in the "safe" place.

File: pandas/core/ops/missing.py [hunk 1]
```diff
@@ -88,8 +88,10 @@ def mask_zero_div_zero(x, y, result):
     Examples
     --------
     >>> x = np.array([1, 0, -1], dtype=np.int64)
+    >>> x
+    array([ 1,  0, -1])
     >>> y = 0       # int 0; numpy behavior is different with float
-    >>> result = x / y
+    >>> result = x // y
```
Comment (simonjayhawkins) [inline pandas/core/ops/missing.py:94]: comment in code: no need to do this for truediv; in py3 numpy behaves the way we want.

maybe update the docstring to reflect this change.

Comment (MomIsBestFriend) [reply pandas/core/ops/missing.py:94]: @simonjayhawkins I'm not sure I understand what to update, can you be more specific?

Comment (simonjayhawkins) [reply pandas/core/ops/missing.py:94]: https://github.com/pandas-dev/pandas/blob/6f93898d32c0f1fdb382d1e9dee434c158998374/pandas/core/ops/missing.py#L73-L76

**Review by simonjayhawkins — APPROVED** [review]
Thanks @MomIsBestFriend lgtm. ping on green

**Comment by MomIsBestFriend** [comment]
ping @simonjayhawkins

**Comment by simonjayhawkins** [comment]
Thanks @MomIsBestFriend

**Pull request merged**